package.json

```diff
@@ -104,7 +104,7 @@
     "strip-json-comments": "^3.0.1"
   },
   "engines": {
-    "node": ">= 6.17.1"
+    "node": ">= 10.16.2"
```

Comment:
Is there a reason not to allow the first 10.x LTS release ([10.13.0](https://nodejs.org/ru/blog/release/v10.13.0/))?

Reply:
No, I change it